include/openssl/core_names.h

```diff
@@ -101,10 +101,22 @@ extern "C" {
 #define OSSL_PKEY_PARAM_DH_PUB_KEY   "dh-pub"
 #define OSSL_PKEY_PARAM_DH_PRIV_KEY  "dh-priv"
 
+/* DSA Parameters */
+#define OSSL_PKEY_PARAM_DSA_P        "dsa-p"
+#define OSSL_PKEY_PARAM_DSA_G        "dsa-g"
+#define OSSL_PKEY_PARAM_DSA_Q        "dsa-q"
```

Comment:
@slontis suggested that these and the DH parameters could be made the same.  See how that ended up in #9670

Reply:
Fixed